Documentation Refactor (#82)
confused-Techie/atom-backend: docs/reference/returns.md

> This document will details the JSON schema that many endpoints are expected to provide, as well as the names assigned to them in this project.

Reviewer: ```suggestion
# Returns

This document details the JSON schema that many endpoints are expected to provide, as well as the names assigned to them in this project.
```

## Package Object Short

> This is the most common return for a package, which leaves out some details of the package, most notably, all versions extra versions and version information. But this includes enough data allowing the user to understand the package itself, and to download the latest version of it's source.

Reviewer: ```suggestion
This is the most common return for a package, which leaves out some details of the package, most notably, all versions extra versions and version information. This includes enough data allowing the user to understand the package itself and to download the latest version of its source.
```
